pkg/monitor/agent/listener/listener.go

```diff
@@ -61,6 +61,6 @@ func IsDisconnected(err error) bool {
 		return false
 	}
 
-	var errn unix.Errno
-	return errors.As(syscerr.Err, &errn) && errors.Is(errn, unix.EPIPE)
+	var errn syscall.Errno
+	return errors.As(syscerr.Err, &errn) && errors.Is(errn, syscall.EPIPE)
```

Comment:
Don't change the error code. This can be moved to a separate file.

Reply:
renamed the file